makefiles/kconfig.mk

```diff
@@ -6,6 +6,7 @@ include $(RIOTMAKE)/tools/kconfiglib.inc.mk
 
 # Generated dir will contain Kconfig generated configurations
 GENERATED_DIR = $(BINDIR)/generated
+KCONFIG_SYNC_DIR = $(GENERATED_DIR)/deps
```

Comment:
I would be good to have some documentation lines for this variable and for `KCONFIG_SYNC_DEPS`.